From 2a115e0f3da54f69c0b4a3cd63cf58bdae4af933 Mon Sep 17 00:00:00 2001
From: "Kothapeta, BikshapathiX" <bikshapathix.kothapeta@intel.com>
Date: Wed, 30 Nov 2022 15:20:21 +0530
Subject: [PATCH] Add mediatranscode sepolicy rules to public.

As mediatranscode update in google moving from private to public

Tracked-On:
Signed-off-by: Kothapeta, BikshapathiX <bikshapathix.kothapeta@intel.com>

diff --git a/groups/media/auto/BoardConfig.mk b/groups/media/auto/BoardConfig.mk
index f348cef..ef36fe8 100644
--- a/groups/media/auto/BoardConfig.mk
+++ b/groups/media/auto/BoardConfig.mk
@@ -1,4 +1,5 @@
 SYSTEM_EXT_PRIVATE_SEPOLICY_DIRS += $(INTEL_PATH_SEPOLICY)/system_ext/private
+SYSTEM_EXT_PUBLIC_SEPOLICY_DIRS += $(INTEL_PATH_SEPOLICY)/system_ext/public
 
 {{#enable_msdk_omx}}
 INTEL_STAGEFRIGHT := true
-- 
2.38.1

